src/foremast/pipeline/create_pipeline.py

```diff
@@ -172,19 +172,29 @@ def get_existing_pipelines(self):
 
         return resp.json()
 
-    def compare_with_existing(self, region='us-east-1'):
+    def compare_with_existing(self, region='us-east-1', onetime=False):
         """Compare desired pipeline with existing pipelines.
 
         Args:
             region (str): Region of desired pipeline.
+            onetime (bool): Looks for different pipeline if Onetime
 
         Returns:
             str: pipeline_id if existing, empty string of not.
         """
         pipelines = self.get_existing_pipelines()
         pipeline_id = None
+        found = False
         for pipeline in pipelines:
-            if (pipeline['application'] == self.app_name) and (region in pipeline['name']):
+            right_app_and_region = (pipeline['application'] == self.app_name) and (region in pipeline['name']) 
```

Comment:
Any opportunity to make this a bit more verbose such as `correct` or `valid`? The right threw me off a bit. :)